### PR TITLE
Update 'uuid' to 0.7.0

### PR DESCRIPTION
**Pull request by kri5**
There is a new release for the uuid crate. This pull request Bumps to the latest release.

### CODE REVIEW

File: contrib/lib/src/uuid.rs [hunk 1]
```diff
@@ -164,6 +164,12 @@ mod test {
     fn test_from_param_invalid() {
         let uuid_str = "c1aa1e3b-9614-4895-9ebd-705255fa5bc2p";
         let uuid_result = Uuid::from_param(uuid_str.into());
-        assert_eq!(uuid_result, Err(UuidParseError::InvalidLength(37)));
+        assert_eq!(
+            uuid_result,
+            Err(UuidParseError::InvalidLength {
+                expected: uuid_ext::parser::Expected::Any(&[36, 32]),
```
Comment (jebrosen) [inline contrib/lib/src/uuid.rs]: A minor version of `uuid` might conceivably change the list of values in `Expected` -- either to include the value 45 for URN format or for a new format it parses. Maybe this assert should be changed or `#[should_panic(expected=...)]` could be used to loosen the expectations for the error.

**Comment by kri5** [comment]
I updated the changes accordinly to your comments.

I'm not sure about the text case, i'm pretty new to rust.

I also changes the version in the examples version to pin it to 0.7.0, the CI was failing for this reason.

If you find anything else, feel free to comment :)

**Comment by jebrosen** [comment]
My thinking with the `should_panic` option was actually to just `Uuid::from_param(...).unwrap()`, which will automatically panic with `uuid`'s `Debug` output. That said, I don't really have a strong opinion on what the test should do -- it's just nice to try to future-proof it a bit for the CI.

**Comment by kri5** [comment]
I updated the test accordingly, i don't know why, first time i tried to this this way, it didn't seem to work, that's why i went the other way around.

**Comment by kri5** [comment]
@SergioBenitez I updated the PR accordingly to your comments

**Comment by SergioBenitez** [comment]
@jebrosen Feel free to rebase and merge this commit with an appropriately stylized message.

**Comment by jebrosen** [comment]
Merged in 7b050eb. Thanks, @kri5!

**Pull request closed without merging**